# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libEGL/egl_loader_autogen.h
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/libEGL/egl_loader_autogen.h
--- a/gfx/angle/checkout/src/libEGL/egl_loader_autogen.h	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/libEGL/egl_loader_autogen.h	Sat Sep 07 13:38:36 2019 +0000
@@ -64,9 +64,13 @@
 extern PFNEGLGETFRAMETIMESTAMPSUPPORTEDANDROIDPROC EGL_GetFrameTimestampSupportedANDROID;
 extern PFNEGLGETFRAMETIMESTAMPSANDROIDPROC EGL_GetFrameTimestampsANDROID;
 extern PFNEGLGETNEXTFRAMEIDANDROIDPROC EGL_GetNextFrameIdANDROID;
+extern PFNEGLGETNATIVECLIENTBUFFERANDROIDPROC EGL_GetNativeClientBufferANDROID;
+extern PFNEGLDUPNATIVEFENCEFDANDROIDPROC EGL_DupNativeFenceFDANDROID;
 extern PFNEGLPRESENTATIONTIMEANDROIDPROC EGL_PresentationTimeANDROID;
 extern PFNEGLCREATEDEVICEANGLEPROC EGL_CreateDeviceANGLE;
 extern PFNEGLRELEASEDEVICEANGLEPROC EGL_ReleaseDeviceANGLE;
+extern PFNEGLQUERYDISPLAYATTRIBANGLEPROC EGL_QueryDisplayAttribANGLE;
+extern PFNEGLQUERYSTRINGIANGLEPROC EGL_QueryStringiANGLE;
 extern PFNEGLPROGRAMCACHEGETATTRIBANGLEPROC EGL_ProgramCacheGetAttribANGLE;
 extern PFNEGLPROGRAMCACHEPOPULATEANGLEPROC EGL_ProgramCachePopulateANGLE;
 extern PFNEGLPROGRAMCACHEQUERYANGLEPROC EGL_ProgramCacheQueryANGLE;
@@ -108,7 +112,7 @@
 {
 using GenericProc = void (*)();
 using LoadProc    = GenericProc(KHRONOS_APIENTRY *)(const char *);
-void LoadEGL(LoadProc loadProc);
+void LoadEGL_EGL(LoadProc loadProc);
 }  // namespace angle
 
 #endif  // LIBEGL_EGL_LOADER_AUTOGEN_H_